pixi.lock

```diff
@@ -2836,10 +2862,10 @@ environments:
       - conda: https://prefix.dev/conda-forge/linux-64/ld_impl_linux-64-2.43-h712a8e2_2.conda
       - conda: https://prefix.dev/conda-forge/linux-64/lerc-4.0.0-h27087fc_0.tar.bz2
       - conda: https://prefix.dev/conda-forge/linux-64/libabseil-20240722.0-cxx17_hbbce691_4.conda
-      - conda: https://prefix.dev/conda-forge/linux-64/libarrow-19.0.0-h9639f6d_3_cuda.conda
-      - conda: https://prefix.dev/conda-forge/linux-64/libarrow-acero-19.0.0-hb826db4_3_cuda.conda
-      - conda: https://prefix.dev/conda-forge/linux-64/libarrow-dataset-19.0.0-hb826db4_3_cuda.conda
-      - conda: https://prefix.dev/conda-forge/linux-64/libarrow-substrait-19.0.0-hbf482d9_3_cuda.conda
+      - conda: https://prefix.dev/conda-forge/linux-64/libarrow-19.0.0-h00a82cf_6_cpu.conda
```

Comment:
Neither jax nor cupy to my knowledge interface with libarrow, so this change looks OK